src/renderer/Core/Search/SearchResultListItem.tsx

```diff
@@ -13,6 +12,7 @@ type SearchResultListItemProps = {
     onClick: () => void;
     onDoubleClick: () => void;
     searchResultItem: SearchResultItem;
+    scrollBehaviour: ScrollBehavior;
```

Comment:
Also here: please stay consistent with naming.

```suggestion
    scrollBehavior: ScrollBehavior;
```